.github/workflows/os-zoo.yml

```diff
@@ -27,17 +28,26 @@ jobs:
       image: docker.io/library/alpine:${{ matrix.tag }}
     env:
       # https://www.openwall.com/lists/musl/2022/02/16/14
-      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare' || '' }}
+      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare  -Wno-covered-switch-default' || '' }}
```

Comment:
I'd try instead to remove the -Wswitch-default from --strict-warnings,
as I did here: https://github.com/bernd-edlinger/openssl/commit/35c23578b98455cfe9eab22529bc6c57efd1b99d

Reply:
hmm, that seems reasonable.  thats odd though, with -Wswitch-default in place there, how are pre clang-18 builds not failing all over the place when strict-warnings are specified?

Reply:
I think this was a non-working warning option that was fixed, so it will hit other users too just a question of when.

Reply:
Given that we are building several branches here, I've just disabled it in CI for now.  The right fix is going to have to be adding default cases in the future I think